app/controllers/reading_list_items_controller.rb

```diff
@@ -3,6 +3,9 @@ def index
     @reading_list_items_index = true
     @reading_collections = ReadingCollection.where(user_id: session_current_user_id).to_json
     # Article.joins(:taggings).joins(:tags).select("articles.*").where("tags.name = 'javascript'")
+    # Article.joins(:tags).where("tags.name = 'javascript'")
+    # Article.joins(:tags).where("tags.name = 'javascript'").where("articles.created_at < ?", 2.days.ago)
+    # Article.joins(:tags).where("tags.name = 'javascript'").where("articles.created_at BETWEEN ? AND ?",Time.now-2.weeks,Time.now)
```

Comment:
Take it these were notes for the collections query, but these are going to be made obsolete by the new taggable process we were looking at?

Reply:
Yup! We actually moved this logic to the ReadingCollection model on Phil's branch. The logic is similar but we learned about the `acts_as_taggable` gem that really helps!